tabs/applications-setup/browser-setup.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/sh -e
+
+. ../common-script.sh
+
+install_chrome() {
+    printf "%b\n" "${YELLOW}Installing Google Chrome..${RC}."
+    case "$PACKAGER" in
+        apt-get|nala)
+            wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+            "$ESCALATION_TOOL" dpkg -i google-chrome-stable_current_amd64.deb
+            ;;
+        zypper)
+            "$ESCALATION_TOOL" "$PACKAGER" addrepo http://dl.google.com/linux/chrome/rpm/stable/x86_64 Google-Chrome
+            "$ESCALATION_TOOL" "$PACKAGER" refresh
+            "$ESCALATION_TOOL" "$PACKAGER" install google-chrome-stable
+            ;;
+        pacman)
+            "$AUR_HELPER" -S google-chrome
+            ;;
+        dnf)
+            "$ESCALATION_TOOL" "$PACKAGER" install fedora-workstation-repositories
+            "$ESCALATION_TOOL" "$PACKAGER" config-manager --set-enabled google-chrome
+            "$ESCALATION_TOOL" "$PACKAGER" install google-chrome-stable
+            ;;
+        *)
+            printf "%b\n" "${RED}The script does not support your Distro. Install manually..${RC}"
+            ;;
+    esac
+
+}
+
+install_firefox() {
+    printf "%b\n" "${YELLOW}Installing Mozilla Firefox...${RC}"
+    case "$PACKAGER" in
+        apt-get)
+            "$ESCALATION_TOOL" "$PACKAGER" update
+            "$ESCALATION_TOOL" "$PACKAGER" install -y firefox
+            ;;
+        zypper)
+            "$ESCALATION_TOOL" "$PACKAGER" in MozillaFirefox
```

Comment:
`install` just to match other instances.

```suggestion
            "$ESCALATION_TOOL" "$PACKAGER" --non-interactive install MozillaFirefox
```